.github/workflows/deploy_beta_testing.yml

```diff
@@ -42,10 +42,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Download war artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4.1.7
```

Comment:
Is there a reason why "upload artifact" is set to "v4" but "download-artifact" is a specific version (v4.1.7)? Should we put a comment above the 4.1.7 lines to explain why it's pinned so specifically?

There's one case here and two more in the other file.

Reply:
As mentioned in an issue that there was a bug in download V4 that was fixed in 4.1.7 https://github.com/actions/upload-artifact/issues/478

Reply:
added comment